src/components/Calculator.js

```diff
@@ -1,61 +1,54 @@
-            <Button classStr="button operator" onBtnClick={onBtnClick} title="x" />
-            <Button classStr="button number" onBtnClick={onBtnClick} title="4" />
-            <Button classStr="button number" onBtnClick={onBtnClick} title="5" />
-            <Button classStr="button number" onBtnClick={onBtnClick} title="6" />
-            <Button classStr="button operator" onBtnClick={onBtnClick} title="-" />
-            <Button classStr="button number" onBtnClick={onBtnClick} title="1" />
-            <Button classStr="button number" onBtnClick={onBtnClick} title="2" />
-            <Button classStr="button number" onBtnClick={onBtnClick} title="3" />
-            <Button classStr="button operator" onBtnClick={onBtnClick} title="+" />
-            <Button classStr="button number zero-btn" onBtnClick={onBtnClick} title="0" />
-            <Button classStr="button number dot" onBtnClick={onBtnClick} title="." />
-            <Button classStr="button operator" onBtnClick={onBtnClick} title="=" />
-          </div>
+  const onBtnClick = (title) => {
+    setObj(calculate(obj, title));
+  };
+  const { total, next, operation } = obj;
+  return (
+    <>
+      <div className="container">
+        <div className="result">
+          { total || '' }
+          { operation || '' }
+          { next || '' }
+        </div>
+        <div className="buttons">
+          <Button classStr="button resetBtn" onBtnClick={onBtnClick} title="AC" />
+          <Button classStr="button negative operator" onBtnClick={onBtnClick} title="+/-" />
+          <Button classStr="button operator percent" onBtnClick={onBtnClick} title="%" />
+          <Button classStr="button operator" onBtnClick={onBtnClick} title="÷" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="7" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="8" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="9" />
+          <Button classStr="button operator" onBtnClick={onBtnClick} title="x" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="4" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="5" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="6" />
+          <Button classStr="button operator" onBtnClick={onBtnClick} title="-" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="1" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="2" />
+          <Button classStr="button number" onBtnClick={onBtnClick} title="3" />
+          <Button classStr="button operator" onBtnClick={onBtnClick} title="+" />
+          <Button classStr="button number zero-btn" onBtnClick={onBtnClick} title="0" />
+          <Button classStr="button number dot" onBtnClick={onBtnClick} title="." />
+          <Button classStr="button operator" onBtnClick={onBtnClick} title="=" />
```

Comment:
- [x] OPTIONAL: This could have been a separate component (`Buttons`) in a separate file and rendered here. Smaller components are easy to fix, reusable, and make the app more scalable. :pray: